### PR TITLE
add flaky test report template

### PR DESCRIPTION
**Pull request by cxzl25**
### _Why are the changes needed?_
Provide report flaky test report template to better track problems.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by cxzl25** [comment]
## Desc

image

## Detail

image

File: .github/ISSUE_TEMPLATE/flaky-test-report.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Kyuubi Flaky Test Report
+title: "Flaky Test: "
+description: Describe the flaky test you encountered with Apache Kyuubi
+labels: ["kind:bug,priority:major"]
```
Comment (yanghua) [inline .github/ISSUE_TEMPLATE/flaky-test-report.yml:21]: Can we have a `test` lable?

Comment (cxzl25) [reply .github/ISSUE_TEMPLATE/flaky-test-report.yml:21]: How about this?
```yml
labels: ["kind:bug,kind:test,priority:major"]
```

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**